### ISSUE TITLE
No input layer spotted

### ISSUE THREAD

**Issue by santle** [tensorflow/models#1420]
My intended task is:
1) create checkpoint files for my custom classes using inception V3 arch.
2) freeze them to the protobuf (.pb) using freeze_graph 
3) optimize the frozen graph using optimize_for_inference
4) Use pb file in android TF camera demo for classification

When I check the .pb file obtained in step 2 for input layer name and output layer name, it shows
	>> bazel-bin/tensorflow/tools/graph_transforms/summarize_graph --in_graph=frozen_model.pb
	No inputs spotted.
	No variables spotted.
	Found 1 possible outputs: (name=tower_0/logits/predictions, op=Softmax) 
	Found 21781804 (21.78M) const parameters, 0 (0) variable parameters, and 188 control_edges
	Op types used: 777 Const, 378 Mul, 284 Add, 283 Sub, 190 Identity, 188 Sum, 96 Reshape, 94 
        Conv2D, 94 StopGradient, 94 SquaredDifference, 94 Square, 94 Mean, 94 Rsqrt, 94 Relu, 94 
        Reciprocal, 15 ConcatV2, 10 AvgPool, 4 MaxPool, 1 RealDiv, 1 RandomUniform, 1 
       QueueDequeueManyV2, 1 Softmax, 1 Split, 1 MatMul, 1 Floor, 1 FIFOQueueV2, 1 BiasAdd

The output layer name is correctly displayed but the input layer name is missing. 
The input layer name is required to use in optimize_for_interface:
	bazel-bin/tensorflow/python/tools/optimize_for_inference \
	--input=frozen_model.pb \
	--output=stripped_graph.pb \
	--input_names= put_your_input_layer_name \
	--output_names=tower_0/logits/predictions

Also, I need to mention in ClassifierActivity.java of the camera demo file:
  private static final String INPUT_NAME = "put_your_input_layer_name";
  private static final String OUTPUT_NAME = "tower_0/logits/predictions";
  private static final String MODEL_FILE = "file:///android_asset/stripped_graph.pb";
  private static final String LABEL_FILE = "file:///android_asset/imagenet_comp_graph_label_strings.txt";
  
 How to include/spot the input layer name.

**Comment by girving** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!

**Comment by santle** [comment]
Ok . The question is moved to stackoverflow. Thanks.
http://stackoverflow.com/questions/43777481/crash-on-porting-inception-v3-custom-model-into-android-tensor-flow-camera-demo

**Comment by lechatthecat** [comment]
I have same problem:
No inputs spotted.
No variables spotted.
Found 1 possible outputs: (name=train_op, op=Identity) 
Found 28833259 (28.83M) const parameters, 0 (0) variable parameters, and 6841 control_edges

And seems like no helpful answer in his question in stackoverflow curremtly.
